app/models/product.rb

```diff
@@ -0,0 +1,3 @@
+class Product < ActiveRecord::Base
+    has_many :sales 
+end
```

Comment:
If you also had the product belonging to the vendor, you could have more concise code in the controllers. See comment around this line: https://github.com/Ada-C5/FarMarRails/pull/11/files#diff-39a5771a121414577ef175c830ffa584R53